lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -633,6 +633,19 @@ radius_server:
     config_set: '<state> radius-server host <ip> key <key>'
     default_value: null
 
+radius_server_group:
+  groups:
```

Comment:
Nit: this is only processing a single group at a time so 'group:' seems appropriate.